.github/workflows/connector-ci-checks.yml

```diff
@@ -86,21 +86,31 @@ jobs:
               - 'airbyte-cdk/java/**/**/src/main/**/*'
               - 'airbyte-cdk/bulk/**/**/src/main/**/*'
 
+      - name: Install airbyte-ops CLI
+        run: |
+          # Install uv if not available
+          if ! command -v uv &> /dev/null; then
+            curl -LsSf https://astral.sh/uv/install.sh | sh
+            export PATH="$HOME/.local/bin:$PATH"
+          fi
+          # Install airbyte-ops CLI tool
+          uv tool install airbyte-internal-ops
+
       - name: Generate Connector Matrix from Changes
         id: generate-matrix
         run: |
-          # Get the list of modified connectors
-          echo "connectors_matrix=$(./poe-tasks/get-modified-connectors.sh --json)" | tee -a $GITHUB_OUTPUT
+          # Get the list of modified connectors using airbyte-ops CLI
+          echo "connectors_matrix=$(airbyte-ops repo connector list --repo-path . --modified-only --output-format json-gh-matrix)" | tee -a $GITHUB_OUTPUT
 
           # If local-cdk-changed is true, get Java connectors with useLocalCdk = true
           # Otherwise, get modified Java connectors
           if [[ "${{ steps.cdk-changes.outputs.java-cdk == 'true' }}" == "true" ]]; then
-            echo "jvm_connectors_matrix=$(./poe-tasks/get-modified-connectors.sh --json --local-cdk)" | tee -a $GITHUB_OUTPUT
+            echo "jvm_connectors_matrix=$(airbyte-ops repo connector list --repo-path . --modified-only --local-cdk --language java --output-format json-gh-matrix)" | tee -a $GITHUB_OUTPUT
```

Comment:
If we assume the working directory is the airbyte repo, then `--repo-path .` is redundant and should be rmeoved.